Makefile

```diff
@@ -1,23 +1,32 @@
-
 SRC = lib/*.js
 
 include node_modules/make-lint/index.mk
 
+BIN = iojs
+
+ifeq ($(findstring io.js, $(shell which node)),)
+	BIN = node
+endif
+
+ifeq (node, $(BIN))
+	FLAGS = --harmony-generators
+endif
+
 TESTS = test/application \
 	test/context/* \
 	test/request/* \
 	test/response/*
 
 test:
-	@NODE_ENV=test ./node_modules/.bin/mocha \
+	@NODE_ENV=test $(BIN) $(FLAGS) \
+		./node_modules/.bin/_mocha \
 		--require should \
-		--harmony-generators \
```

Comment:
Since it is not required for io.js and so the majority of users won't run io.js with this flag, it ensures that the test suite is fully compliant.